actors/builtin/storage_miner/storage_miner_actor.go

```diff
@@ -402,25 +421,35 @@ type ExtendSectorExpirationParams struct {
 }
 
 func (a *StorageMinerActor) ExtendSectorExpiration(rt Runtime, params *ExtendSectorExpirationParams) *adt.EmptyValue {
-	storageWeightDescPrev := a._rtGetStorageWeightDescForSector(rt, params.sectorNumber)
+	sectorNo := params.sectorNumber
```

Comment:
:thinking: this just a style thing?

Reply:
For brevity, since it's used so many times below.